site/en/docs/devtools/open/index.md

```diff
@@ -61,3 +62,10 @@ Chrome.
 
 [1]: /docs/devtools/css
 [2]: /docs/devtools/console
+
+
+## What's next?
+
+Congratulations! You've successfully unlocked the power of Chrome DevTools. Next, watch the following video to learn some useful shortcuts and settings for quicker DevTools navigation.
+
+{% YouTube id="H0XScE08hy8" %}
```

Comment:
Is that the right video? It's about debugging, not shortcuts and settings.

Reply:
Ooops, it was not. Fixed.